### DISCUSSION TITLE
V2 - Terp-Core

### DISCUSSION THREAD

**Discussion by discoverdefiteam** [terpnetwork/terp-core#92] [General]
# TODO

- [ ] migrate from wasmd fork
- [ ] format upgrade folder
- [ ] Add packet forwarding middleware
- [ ] Add Global Configs
- [ ] Add FeeShare Mechanics
- [ ] Add IBC Hooks
- [ ] Update Integration Testing 
- [ ] Update repo scripts

**Comment by hard-nett** [comment]
Started [`migrate from wasmd fork`](https://github.com/hard-nett/terp-core/tree/v2.0)
